diff --git a/description/urdf/hokuyo.urdf.xacro b/description/urdf/hokuyo.urdf.xacro
index 4a1767f..898a656 100644
--- a/description/urdf/hokuyo.urdf.xacro
+++ b/description/urdf/hokuyo.urdf.xacro
@@ -55,7 +55,7 @@
                         <resolution>0.001</resolution>
                     </range>
                     <noise>
-                        <type>Gaussian</type>
+                        <type>gaussian</type>
                         <mean>0.0</mean>
                         <stddev>0.01</stddev>
                     </noise>
diff --git a/launch/world-park.launch b/launch/world-park.launch
index 3b50ab1..995a06f 100644
--- a/launch/world-park.launch
+++ b/launch/world-park.launch
@@ -8,12 +8,12 @@
   <arg name="gui" default="true"/>
   <arg name="simple_rviz" default="true"/>
   <arg name="headless" default="false"/>
-  <arg name="debug" default="false"/>
+  <arg name="debug" default="true"/>
   <arg name="verbose" default="true"/>
 
   <!-- Set the type of terrain-->
-<!--  <arg name="scenario" default="flatMinefield"/>-->
-  <arg name="scenario" default="softWavyMinefield"/>
+ <arg name="scenario" default="flatMinefield"/>
+  <!-- <arg name="scenario" default="softWavyMinefield"/> -->
 <!--  <arg name="scenario" default="hardWavyMinefield"/>-->
 
   <arg name="world" default="$(find hratc2017_framework)/description/worlds/$(arg scenario).world" />
@@ -34,7 +34,9 @@
   <arg     if="$(arg debug)" name="script_type" value="debug"/>
 
   <!-- start gazebo server-->
-  <node name="gazebo" pkg="gazebo_ros" type="$(arg script_type)" respawn="false" output="screen"
+  <!-- <node launch-prefix="$(find debug_)/scripts/tmux_gdb" name="gazebo" pkg="gazebo_ros" type="$(arg script_type)" respawn="false" output="screen" -->
+  <!-- <node launch-prefix="$(find gazebo_worlds)/scripts/gdbrun" name="gazebo" pkg="gazebo_ros" type="$(arg script_type)" respawn="false" output="screen" -->
+  <node  name="gazebo" pkg="gazebo_ros" type="$(arg script_type)" respawn="false" output="screen"
 	args="$(arg command_arg1) $(arg command_arg2) $(arg command_arg3) $(arg world)">
     <remap from="tf" to="gazebo_tf"/>
   </node>
diff --git a/settings/minefield.yaml b/settings/minefield.yaml
index cdece7f..a5bcacf 100644
--- a/settings/minefield.yaml
+++ b/settings/minefield.yaml
@@ -1,36 +1,21 @@
 minefield:
 
        corner1: 
-          latitude: 11.364583
-          longitude: 4.4797
+          latitude: 0
+          longitude: 0
           altitude: 10
 
        corner2: 
-          latitude: -11.320282
-          longitude: 3.519631
+          latitude: 12.0
+          longitude: 0.0
           altitude: 10
 
        corner3: 
-          latitude: -10.899889
-          longitude: -1.351829
+          latitude: 12.0
+          longitude: 12.0
           altitude: 10
 
        corner4: 
-          latitude: -4.629055
-          longitude: -0.81907
+          latitude: 0.0
+          longitude: 12.0
           altitude: 10
-
-       corner5: 
-          latitude: -1.277216
-          longitude: -1.812646
-          altitude: 10
-
-       corner6: 
-          latitude: 5.920414
-          longitude: -1.983078
-          altitude: 10
-
-       corner7: 
-          latitude: 10.841441
-          longitude: -2.032711999
-          altitude: 10
\ No newline at end of file
